packages/authorization-process/src/model/domain/models.ts

```diff
@@ -11,3 +11,4 @@ export type ApiKeySeed = z.infer<typeof api.schemas.KeySeed>;
 export type ApiPurposeAdditionSeed = z.infer<
   typeof api.schemas.PurposeAdditionDetails
 >;
+export type JWKKey = z.infer<typeof api.schemas.JWKKey>;
```

Comment:
```suggestion
export type ApiJWKKey = z.infer<typeof api.schemas.JWKKey>;
```
We usually refer to these as Api* since it's the model we expose